apps/meteor/app/assets/server/assets.ts

```diff
@@ -301,23 +321,25 @@ export const RocketChatAssets = new (class {
 		return assetValue.cache;
 	}
 
-	getURL(assetName, options = { cdn: false, full: true }) {
-		const asset = settings.get(assetName);
+	public getURL(assetName: string, options = { cdn: false, full: true }): string {
+		const asset = settings.get<Record<string, any>>(assetName);
```

Comment:
this returns a `IRocketChatAsset` right?
```suggestion
		const asset = settings.get<IRocketChatAsset>(assetName);
```